README.md

```diff
@@ -21,7 +21,7 @@ Codeship Pro | [ ![Codeship Pro CI](https://app.codeship.com/projects/8d6a20c0-b
 GitLab | [![GitLab CI](https://gitlab.com/cypress-io/cypress-example-kitchensink/badges/master/pipeline.svg)](https://gitlab.com/cypress-io/cypress-example-kitchensink/commits/master) | [basic/.gitlab-ci.yml](basic/.gitlab-ci.yml) | [.gitlab-ci.yml](.gitlab-ci.yml)
 Heroku CI | | [basic/app.json](basic/app.json) |
 Jenkins | | [basic/Jenkinsfile](basic/Jenkinsfile) | [Jenkinsfile](Jenkinsfile)
-Semaphore v2 | [![Semaphore CI](https://semaphoreci.com/api/v1/cypress-io/cypress-example-kitchensink/branches/master/badge.svg)](https://semaphoreci.com/cypress-io/cypress-example-kitchensink) | [.semaphore/semaphore.yml](.semaphore/semaphore.yml)
+Semaphore v2 | [Project dashboard](https://cypress-io.semaphoreci.com/projects/cypress-example-kitchensink) | [basic/.semaphore.yml](basic/.semaphore.yml) | [.semaphore/semaphore.yml](.semaphore/semaphore.yml)
```

Comment:
Shouldn't you link Semaphore v1 here too?

Reply:
but since Semaphore v1 instructions are invisible to people, I think we should just retire it